### PR TITLE
Add return crn attribute

### PR DESCRIPTION
**Pull request by Diptipowervs**
This in response to: https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5061

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_workspace.go [hunk 1]
```diff
@@ -121,6 +122,11 @@ func resourceIBMPIWorkspaceRead(ctx context.Context, d *schema.ResourceData, met
 		return diag.FromErr(err)
 	}
 	d.Set(PIWorkspaceName, controller.Name)
+	wsDetails := map[string]interface{}{
+		WorkspaceCreationDate: controller.CreatedAt,
+		WorkspaceCRN:          controller.CRN,
```
Comment (michaelkad) [inline ibm/service/power/resource_ibm_pi_workspace.go:127]: This should be the targetcrn double check that when you ran it
```suggestion
		WorkspaceCRN:          controller.TargetCRN,
```

**Review by michaelkad — CHANGES_REQUESTED** [review]
You have to update the doc as well

File: website/docs/d/pi_workspace.html.markdown [hunk 2]
```diff
@@ -55,7 +55,7 @@ In addition to all argument reference listed, you can access the following attri
 
     Nested schema for `pi_workspace_details`:
   - `creation_date` - (String) Workspace creation date.
-  - `crn` - (String) Workspace crn.
+  - `crn` - (String) Workspace Target crn.
```
Comment (michaelkad) [inline website/docs/d/pi_workspace.html.markdown:58]:
```suggestion
  - `crn` - (String) Workspace  crn.
```

Comment (michaelkad) [reply website/docs/d/pi_workspace.html.markdown:58]: The description doesn't need to change.

**Review by michaelkad — CHANGES_REQUESTED** [review]
One last comment!

**Review by michaelkad — DISMISSED** [review]

**Comment by ismirlia** [comment]
This has been merged in the main terraform repo closing...

**Pull request closed without merging**